README.md

```diff
@@ -1,21 +1,21 @@
 ## SourceCred
 
-SourceCred is a system for creating digital tokens, called 'Cred', for open-source projects. Cred represents credit for doing the work of developing and maintaining the project. For example, people who contribute to SourceCred's development will receive Cred in SourceCred itself.
+SourceCred is a system for creating digital tokens, called 'Cred' (¤), for open-source projects. Cred represents credit for doing the work of developing and maintaining the project. For example, people who contribute to SourceCred's development will receive SourceCred¤.
 
 We want to build tools for fairly distributing Cred to contributors, and incentives that act as rewards for having Cred. We want to reward open-source developers for their work. The world benefits tremendously from open-source, and if its developers were fairly rewarded, it would benefit so much more.
```

Comment:
Perhaps switch these two sentences? something like: 
We want to reward open-source developers for their work, because the world benefits tremendously from open-source, and if its developers were fairly rewarded, it would benefit so much more. To that end, we want to build tools for fairly distributing Cred to contributors, as well as incentives that act as rewards for having Cred.